specification/securityinsights/resource-manager/readme.md

```diff
@@ -26,7 +26,7 @@ These are the global settings for the SecurityInsights API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2023-11
+tag: package-preview-2023-10
```

Comment:
Why is default API Version is being changed from a stable version to preview version here?